pftp/client_handler.go

```diff
@@ -84,7 +84,6 @@ func (c *clientHandler) setClientDeadLine(t int) {
 }
 
 func (c *clientHandler) handleCommands() error {
-	proxyError := make(chan error)
```

Comment:
I can't understand why do you delete `proxyError`?

Reply:
Oh, it just missed.

I tested a few things what channels makes hang sometimes.
But after end of test, I forgot this channel was deleted.

Thanks for find that!

Reply:
reverted : https://github.com/pyama86/pftp/pull/31/commits/a4f90b5b2aa4e270f8963be563c9fc7f99d4a5e2